code/datums/components/storage/storage.dm

```diff
@@ -178,7 +178,8 @@
 
 	if(!isitem(O) || !click_gather || SEND_SIGNAL(O, COMSIG_CONTAINS_STORAGE))
 		return FALSE
-	. = COMPONENT_NO_ATTACK
+	if(!istype(O, /obj/item/slimecross/reproductive))
+		. = COMPONENT_NO_ATTACK
```

Comment:
I dont like that this is done on all storage things for everything for a reproductive override. Can you find where preattack_intercept is called from and override it on the slimecross instead?

Reply:
(override pre_attack on the reproductive extract before ..())